src/settings/Settings.tsx

```diff
@@ -184,6 +184,34 @@ export interface IFeature extends Omit<IBaseSetting<boolean>, "isFeature"> {
 export type ISetting = IBaseSetting | IFeature;
 
 export const SETTINGS: {[setting: string]: ISetting} = {
+    "feature_video_rooms": {
+        isFeature: true,
+        labsGroup: LabGroup.Rooms,
+        displayName: _td("Video rooms"),
+        supportedLevels: LEVELS_FEATURE,
+        default: false,
+        // Reload to ensure that the left panel etc. get remounted
+        controller: new ReloadOnChangeController(),
+        betaInfo: {
+            title: _td("Video rooms"),
+            caption: () => <>
+                <p>{ _t("A new way to chat over voice and video in Element.") }</p>
+                <p>{ _t("Video rooms are always-on VoIP channels embedded within a room in Element.") }</p>
+            </>,
+            faq: () =>
+                SdkConfig.get().bug_report_endpoint_url && <>
+                    <h4>{ _t("How can I create a video room?") }</h4>
+                    <p>{ _t("Use the “+” button in the room section of the left panel.") }</p>
+                    <h4>{ _t("Can I use text chat alongside the video call?") }</h4>
+                    <p>{ _t("Yes, the chat timeline is displayed alongside the video.") }</p>
```

Comment:
The threads beta doesn't warn about that either, nor do other settings such as language that refresh the app. @gaelledel Do you have opinions about this / could you provide copy?

Reply:
Spaces did, Threads really should, a refresh is jarring and unexpected

Reply:
Perhaps we can have a subtext under the CTA saying: "Joining beta will reload the Element app" rather than having it in the text button as it may have been done in the past with spaces